index.js

```diff
@@ -5,15 +5,18 @@ const fp = require('fastify-plugin')
 const Next = require('next')
 
 function fastifyNext (fastify, options, next) {
-  const { underPressure, noServeAssets, ...nextOptions } = options
-
+  const { underPressure, noServeAssets, hostname, port, ...nextOptions } = options
   if (underPressure) {
     const opts = typeof underPressure === 'object' ? underPressure : Object.create(null)
 
     fastify.register(require('@fastify/under-pressure'), opts)
   }
 
-  const app = Next(Object.assign({}, { dev: process.env.NODE_ENV !== 'production' }, nextOptions))
+  const app = Next(Object.assign({}, {
```

Comment:
After some though, the developer can pass `hostname` and `port` directly to the options.
So, when we are helping them to do it. It means that they will never know it is a requirement.

Does middle-ware work when they pass the option through 
```
fastify.register(fastifyNext, {
  hostname: 'localhost',
  port: 3000
})
```